lib/User/User.php

```diff
@@ -170,8 +170,11 @@ public function processAttributes($ldapEntry) {
 
 		//Email
 		$attr = strtolower($this->connection->ldapEmailAttribute);
+		$quotaDefault = $this->connection->ldapQuotaDefault;
 		if(isset($ldapEntry[$attr])) {
 			$this->updateEmail($ldapEntry[$attr][0]);
+		} else if (!empty($quotaDefault)) {
```

Comment:
⁉️ 

I expect to update the quota inside the quota section, not in the email section.